spec/buffered-process-spec.coffee

```diff
@@ -117,6 +111,7 @@ describe "BufferedProcess", ->
       expect(stdout).toEqual ''
 
   it "calls the specified stdout callback only with whole lines", ->
+    # WINSPEC: Fails because command is too long, /bin/echo is *nix only, odd newlining and quoting
```

Comment:
I am not really familiar with the tools that Windows provides which we might use in this case, but maybe we could have a fixture script/exe? Something that we can execute like the following:

```bash
$ node fixtures/echo.js
```